### PR TITLE
[17.0][MIG] mail_outbound_static

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

**Review by amkarthik — APPROVED** [review]
Code review 👍🏾

**Review by atchuthan — APPROVED** [review]
code review :+1:

**Comment by CasVissers-360ERP** [comment]
I keep getting this warning in the log? 

 XML declarations in HTML module descriptions are deprecated since Odoo 17, mail_outbound_static can just have a UTF8 description with not need for a declaration.

**Comment by SodexisTeam** [comment]
> I keep getting this warning in the log?
> 
> XML declarations in HTML module descriptions are deprecated since Odoo 17, mail_outbound_static can just have a UTF8 description with not need for a declaration.

@CasVissers-360ERP 
This recent commit https://github.com/odoo/odoo/commit/9429986b63a8eddc136c7ae5ecca323015155de2 impacted this deprecated warning. It has been fixed. Please check now.

**Review by CasVissers-360ERP — APPROVED** [review]
Running on live migrated v17 db.

**Review by atchuthan — APPROVED** [review]

File: mail_outbound_static/readme/ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (francesco-ooops) [inline mail_outbound_static/readme/ROADMAP.md]: This can be removed i guess

Comment (SodexisTeam) [reply mail_outbound_static/readme/ROADMAP.md]: > This can be removed i guess

@francesco-ooops Removed.

**Comment by yajo** [comment]
Does this module make sense anymore? We already have from filters upstream since 16.0. Check out: https://www.odoo.com/documentation/16.0/applications/general/email_communication/email_servers.html#use-a-default-from-email-address

**Comment by matteoopenf** [comment]
> Does this module make sense anymore? We already have from filters upstream since 16.0. Check out: https://www.odoo.com/documentation/16.0/applications/general/email_communication/email_servers.html#use-a-default-from-email-address

some times this module is nedded in case the mail server is provide by Microsoft with office 365

**Comment by atchuthan** [comment]
> Does this module make sense anymore? We already have from filters upstream since 16.0. Check out: https://www.odoo.com/documentation/16.0/applications/general/email_communication/email_servers.html#use-a-default-from-email-address

This module is also used for multi company purposes too

**Review by dreispt — APPROVED** [review]

**Pull request merged**